umh-core/pkg/sentry/sentry.go

```diff
@@ -0,0 +1,82 @@
+package sentry
+
+import (
+	"os"
+	"strconv"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/getsentry/sentry-go"
+	"go.uber.org/zap"
+)
+
+// InitSentry initializes sentry with the given app name and version
+func InitSentry(appVersion string) {
+
+	environment := "development"
+	sampleRate := 0.3
+
+	version, err := semver.NewVersion(appVersion)
+	if err != nil {
+		zap.S().Errorf("Failed to parse app version, using default environment (development): %s", err)
+	} else {
+		if version.Prerelease() == "" {
+			environment = "production"
+			sampleRate = 0.1
+		}
+	}
+
+	if rate := os.Getenv("SENTRY_SAMPLE_RATE"); rate != "" {
+		if parsedRate, err := strconv.ParseFloat(rate, 64); err == nil {
+			sampleRate = parsedRate
+		}
+	}
+
+	err = sentry.Init(sentry.ClientOptions{
+		Dsn:              "https://abc@staging.management.umh.app/sentry",
```

Comment:
staging?

Reply:
Yes, our apps always seem to use staging.management.umh.app